### PR TITLE
Feature/no train

### PR DESCRIPTION
**Pull request by ewsheng**

### CODE REVIEW

File: src/lib/json-to-api-write-stream.coffee [hunk 1]
```diff
@@ -62,7 +70,22 @@ module.exports = JsonToApiWriteStream = (options = {}) ->
     pipeline.emit 'error', err
 
   # highland still uses 'end' event
-  finishStream.on 'finish', -> pipeline.emit 'finish'
+  finishStream.on 'finish', ->
```
Comment (swarajban) [inline src/lib/json-to-api-write-stream.coffee]: maybe move this into a function so we know exactly what's going on?

**Comment by swarajban** [comment]
made one comment but looks good. Tests are failing tho

**Comment by ewsheng** [comment]
I fixed the tests so that only the ones that were failing before are still failing.

**Comment by swarajban** [comment]
cool looks good

**Pull request merged**